samples/bluetooth/central_gatt_write/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  name: Bluetooth Central HR
```

Comment:
should this be updated (names n stuff)? same comment for the peripheral yaml

Reply:
Done